intel-sgx/ias/src/client.rs

```diff
@@ -64,36 +60,20 @@ pub struct IasVerificationResult {
     pub advisory_ids: Vec<IasAdvisoryId>,
 }
 
-impl From<IasVerificationResult> for AttestationEmbeddedIasReport<'static, 'static, 'static, Unverified> {
-    fn from(res: IasVerificationResult) -> AttestationEmbeddedIasReport<'static, 'static, 'static, Unverified> {
-        let IasVerificationResult {
-            raw_report,
-            signature,
-            cert_chain,
-            ..
-        } = res;
-        AttestationEmbeddedIasReport::from(
-            sgx_pkix::attestation::AttestationEmbeddedIasReport {
-                http_body: raw_report.into(),
-                report_sig: signature.into(),
-                certificates: cert_chain.into_iter().map(Into::into).collect()
-            }
-        )
+impl IasVerificationResult {
+    pub fn report<C: Crypto>(&self, ca_certificates: &[&[u8]]) -> Result<VerifyAttestationEvidenceResponse> {
```

Comment:
`fn verify` instead? Also add a doc comment what you're actually verifying

Reply:
Fixed